README.md

```diff
@@ -7,8 +7,7 @@
 A highly scalable skeleton with best practices, fullstack monorepo containing our front end and backend boilerplates.
 
 - [EOSCR Boilerplate](#eoscr-boilerplate)
-- [Version](#version)
-- [Features!](#features)
+- [Features](#features)
 - [Installation](#installation)
   - [Before to Start](#before-to-start)
```

Comment:
Before to start is not grammatically correct 

lets replace with "Getting Started"